README.md

```diff
@@ -19,10 +19,8 @@ For more detailed information about using the QGreenland Core zip package and on
 how to contribute to QGreenland, see our
 [Documentation](https://qgreenland.readthedocs.io)
 
-> :tada: QGreenland v3 development is well underway. We're looking for testers to
-> provide feedback by **August 6, 2023**. Please see our
-> [alpha release announcement](https://github.com/nsidc/qgreenland/discussions/694)
-> for more details, including highlights of this release.
+> :tada: QGreenland v3 has been released! Please visit our
+> [website](https://qgreenland.org/download) to download it now!
```

Comment:
This will change to a getting started page that then links directly to the download in the new website. Maybe we should just link to the package here instead?

Reply:
I'm indifferent here. If they're on GitHub, they may want to go directly to the package. But if we link to the website, which links to the package, we only need to keep the package link updated in one place.